lib/gitlib.sh

```diff
@@ -24,6 +24,7 @@ GHCURL="curl -s --fail --retry 10 -u ${GITHUB_TOKEN:-$FLAGS_github_token}:x-oaut
 JCURL="curl -g -s --fail --retry 10"
 K8S_GITHUB_API='https://api.github.com/repos/kubernetes/kubernetes'
 K8S_GITHUB_URL='https://github.com/kubernetes/kubernetes'
+K8S_GITHUB_SSH='git@github.com:kubernetes/kubernetes.git'
```

Comment:
In both cases it checks out to `kubernetes/`. I can remove the `.git` if you want to be more consistent with the https URL, though the `.git` suffix is what GitHub supplies if you click the "Clone or Download" button on the repo's page.